### PR TITLE
Free cad gui recipe

### PR DESCRIPTION
**Pull request by Lordling**

### CODE REVIEW

**Comment by Lordling** [comment]
I fixed the one issue pointed out by the conda-smithy linter but it still registers as failed ... any way to update or 'refresh' the check?

**Comment by marcelotrevisani** [comment]
@conda-forge-linter

File: recipes/freecadnoui/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "freecadnoui" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Lordling/FreeCADNoUI/archive/freecadnoui.tar.gz
```
Comment (marcelotrevisani) [inline recipes/freecadnoui/meta.yaml]: I just saw that you don't have a release, please generate one to use here

Comment (marcelotrevisani) [reply recipes/freecadnoui/meta.yaml]: Well.. I just saw that you add a ``.tar.gz`` file in your git project. Would be good to just let your code there and generate tag release to the version which you want to

**Comment by Lordling** [comment]
I generated a release but regardless of the version number I use (v1.0 or the commit tag) the linter fails again. What am I supposed to put in the meta.yaml?

**Pull request closed without merging**